CODEOWNERS

```diff
@@ -55,12 +56,9 @@
 /xwf/ @magma/approvers-xwf
 /feg/radius @magma/approvers-xwf @magma/approvers-cwf
 
-/src/go/ @markjen
-
 /dp/ @magma/approvers-domain-proxy
 
 /docs/ @magma/approvers-docs
-/docs/readmes/proposals @electronjoe
```

Comment:
@electronjoe is still with the project today, but we might want to redistribute this to a group of people non-the-less.
Are there any better candidates than just the fallback @magma/approvers-docs ?

Reply:
I think this needs to be under control of someone in charge of the proposal process. Maybe that needs its own approvers group. Docs is not the right fallback

Reply:
Yes we do want to move this off of a single person (me). approvers-tsc sounds like a  good strategy.